demo/authz.rego

```diff
@@ -0,0 +1,16 @@
+package authz
+
+import future.keywords.if
+import future.keywords.in
+
+default allow := false
+
+deny if {
+	not "admin" in input.user.roles
+}
+
+bad_deny if {
+	"admin" != input.user.roles[_]
```

Comment:
Regal linter error: `not-equals-in-loop`, **[Use of != in loop](https://docs.styra.com/regal/rules/bugs/not-equals-in-loop)**, on column 10 of this line.